string name
float64 goal_x
float64 goal_y
---
float64 x
float64 y